src/components/ViewDeck/ViewDeck.js

```diff
@@ -77,16 +77,23 @@ class ViewDeck extends Component {
   };
 
   parseCards = (cards) => {
-    var allParsedCards = [];
-    cards.forEach((card) => {
-      var parsedCard = JSON.parse(card.api_data);
-      allParsedCards.push(parsedCard);
-    });
-    return allParsedCards;
+    // var allParsedCards = [];
+    // cards.forEach((card) => {
+    //   var parsedCard = JSON.parse(card.api_data);
+    //   allParsedCards.push(parsedCard);
+    // });
+    // return allParsedCards;
+    return cards.map(card => {
+      const jsonData = JSON.parse(card.api_data);
+      return {
+        ...card,
+        jsonData,
+      };
+    })
```

Comment:
I saved your old logic and added my new logic. You could also spread jsonData into the return so that it isn't nested. That is up to you.